app/controllers/omniauth_callbacks_controller.rb

```diff
@@ -44,15 +50,6 @@ def callback_for(provider)
     )
 
     if user_persisted_and_valid?
-      # Deleting the session cookie with the previous app domain, the one without the leading dot.
-      # This should fix the double cookie scenario
-      # NOTE: this code is a hotfix, and shall be removed soon (around 2 weeks from deployment)
-      domain = Rails.env.production? ? ApplicationConfig["APP_DOMAIN"] : nil
-      if domain&.starts_with?(".")
```

Comment:
This didn't work in #10169 because the app domain was `dev.to`, so we never actually made it into this conditional!